modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStore.java

```diff
@@ -418,6 +424,14 @@ public void setFetchSize(int fetchSize) {
         this.fetchSize = fetchSize;
     }
 
+    public int getBatchInsertSize() {
```

Comment:
The javadoc will have to be much more than that, and explain the consequences of batching to those that are not aware of it, like, not being able to immediately see the effects of an insert because it's stuck in the batch queue, and still hasn't hit the database.